mmengine/visualization/vis_backend.py

```diff
@@ -20,14 +20,14 @@
 
 def force_init_env(old_func: Callable) -> Any:
     """Those methods decorated by ``force_init_env`` will be forced to call
-    ``_init_env`` if the instance has not been fully initiated. This function
+    ``_init_env`` if the instance has not been fully initiated.
+
+    This function
     will decorated all the `add_xxx` method and `experiment` method, because
     `VisBackend` is initialized only when used its API.
```

Comment:
```suggestion
    This function will decorated all the `add_xxx` method and `experiment` method, because
    `VisBackend` is initialized only when used its API.
```